compositor_pipeline/src/pipeline/output.rs

```diff
@@ -54,13 +56,17 @@ pub struct RawVideoOptions {
 #[derive(Debug, Clone)]
 pub struct RawAudioOptions;
 
+pub struct KeyframeRequest;
+
 pub enum Output {
     Rtp {
         sender: RtpSender,
         encoder: Encoder,
+        keyframe_req_sender: Sender<KeyframeRequest>,
     },
     EncodedData {
         encoder: Encoder,
+        keyframe_req_sender: Sender<KeyframeRequest>,
```

Comment:
It can be just
```suggestion
        keyframe_req_sender: Sender<()>,
```
but I wanted to avoid passing any other sender/receiver unintentionally.
After compilation, it should behave same as `Sender<()>`